3rdparty/python/requirements.txt

```diff
@@ -36,5 +36,7 @@ pyjwt
 pytest
 requests
 redis
+setuptools>=50.3.0,<57.0
 typing_extensions
+wheel<0.37,>=0.35.1
```

Comment:
I believe Thomas and Chris were discussing `setuptools` recently in the context of requirements.txt, maybe check with them and see if this is needed

Reply:
I've pulled in those changes via rebase.